README.md

```diff
@@ -1,8 +1,83 @@
 # BitmovinPlayerCore
 
-**This package is currently under active development and is not yet ready for production use.**
+- [Installation](#installation)
+    - [CocoaPods](#using-cocoapods)
+    - [Swift Package Manager](#using-swift-package-manager)
+    - [Add the Player SDK to the Project directly](#add-the-player-sdk-to-the-project-directly)
+- [Getting Started](#getting-started)
+    - [Configure the license key](#configure-the-license-key)
+- [Samples](#samples)
+- [Documentation and Resources](#documentation-and-resources)
+
+The `BitmovinPlayerCore` package is the standalone `Player` without additional dependencies to other modules such as `BitmovinAnalytics`.
+
+## Installation
+To add the `BitmovinPlayerCore` SDK as a dependency to your project, you have three options: Using CocoaPods, Swift Package Manager or adding the SDK bundle directly.
+
+### Using [CocoaPods](https://cocoapods.org/)
+Add the following lines to the `Podfile` of your project and replace the `Version Number` with the desired version of the `BitmovinPlayerCore`. You can find all versions [here](https://github.com/bitmovin/cocoapod-specs/tree/master/Specs/BitmovinPlayerCore).
+
+Execute `pod repo update` first to add the new source and execute `pod install` after to install the new `BitmovinPlayerCore` dependency.
+
+```ruby
+source 'https://github.com/bitmovin/cocoapod-specs.git'
+
+pod 'BitmovinPlayerCore', '3.37.1'
+```
+
+### Using [Swift Package Manager](https://swift.org/package-manager/)
+[Swift Package Manager](https://swift.org/package-manager/) is a tool for managing the distribution of Swift frameworks. It integrates with the Swift build system to automate the process of downloading, compiling, and linking dependencies.
+
+#### Using Xcode
+To integrate using Xcode 13, open your Project file and specify it in `Project > Package Dependencies` using the following URL:
+
+```
+https://github.com/bitmovin/player-ios-core.git
+```
+
+#### Using `Package.swift`
+To integrate using Apple's Swift Package Manager, add the following as a dependency to your `Package.swift` and replace `Version Number` with the desired version of the SDK.
+
+```swift
+.package(name: "BitmovinPlayerCore", url: "https://github.com/bitmovin/player-ios-core.git", .exact("Version Number"))
+```
+
+And then specify the `BitmovinPlayerCore` as a dependency of the desired target. Here is an example of a `Package.swift` file:
+
+```swift
+let package = Package(
+  ...
+  dependencies: [
+    ...
+    .package(name: "BitmovinPlayerCore", url: "https://github.com/bitmovin/player-ios.git", .exact("Version Number"))
+  ],
+  targets: [
+    .target(name: "<NAME_OF_YOUR_PACKAGE>", dependencies: ["BitmovinPlayerCore"])
```

Comment:
In this case we have to use this for proper access,
```suggestion
    .target(name: "<NAME_OF_YOUR_PACKAGE>", dependencies: [.product(name: "BitmovinPlayerCore", package: "player-ios-core")])
```

Reply:
[19b83fd](https://github.com/bitmovin/player-ios-core/pull/1/commits/19b83fd7bd72bb31b0dc0146951cf4c9b07a7188)